frappe/tests/test_utils.py

```diff
@@ -510,3 +510,16 @@ def test_link_titles_on_getdoc(self):
 		todo.delete()
 		user.delete()
 		prop_setter.delete()
+
+
+class FrappeTestCase(unittest.TestCase):
+	"""Base test class for Frappe tests."""
+	@classmethod
+	def setUpClass(cls) -> None:
+		frappe.db.commit()
+		return super().setUpClass()
+
+	@classmethod
+	def tearDownClass(cls) -> None:
+		frappe.db.rollback()
+		return super().tearDownClass()
```

Comment:
this looks familiar

![image](https://user-images.githubusercontent.com/9079960/155969052-2a1a8ae6-af62-4ecb-9a46-0206a7f8ef12.png)

Reply:
Yep. Lifted from https://github.com/frappe/erpnext/blob/develop/erpnext/tests/utils.py#L17.

ERPNext should extend and use it. Probably all other frappe apps can do the same.